tensorflow/contrib/go/graph_test.go

```diff
@@ -0,0 +1,252 @@
+package tensorflow_test
+
+import (
+	"testing"
+
+	tf "github.com/tensorflow/tensorflow/tensorflow/contrib/go"
+)
+
+func TestGraphGeneration(t *testing.T) {
+	graph := tf.NewGraph()
+	input1 := graph.Placeholder("input1", tf.DtInt32, []int64{3}, []string{})
```

Comment:
yes, is more correct to use DT... since this is an acronym: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
I'm going to change them